### DISCUSSION TITLE
collegarsi da remoto

### DISCUSSION THREAD

**Discussion by efestopini** [indomus/forum#2785] [HUB: Home Assistant]
buonasera, non essendo esperto, neanche vagamente pratico, volevo chiedere )visto che avevo già fatto un errore simile), per non buttare via ore, se per connettermi da remoto posso seguire la guida che c'è qui sul sito, anche se è espressamente detto che è per linux debian, io giro con ubuntu, incontrerò dei problemi?

**Comment by snakuzzo** [comment]
È praticamente uguale, anche se io sto andando ad intuito, visto che non hai detto che tipo di installazione hai

**Comment by efestopini** [comment]
Non essendo pratico non so dirti, la guida dice espressamente che è valida per ogni tipo di installazione, il mio unico dubbio, visto che in un altra discussione che avevo aperto mi è stato detto esplicitamente " la guida è per linux Debian non per linux ubuntu" facendomi capire che èscontato il fatto che non sia funzionante, volevo sapere se seguendo questa mi ritroverò con qualche comando differente

**Reply by snakuzzo** [reply]
1. non creare nuovi thread ma fai reply sotto l'ultima risposta
2. Hai letto cosa ho scritto? (a pelle, direi di no). Ancora non hai specificato di cosa stai parlando...di quale guida ???

**Reply by efestopini** [reply]
Chiedo umilmente scusa, pensavo che con il titolo che ho messo "collegarsi da remoto" fosse sufficiente a far capire che facevo riferimento alla guida che titola così. Cerco di spiegarmi meglio, ho una installazione di home assistant core su docker su Linux ubuntu, volendo collegarmi da remoto vorrei seguire la guida qui si indomus che intitola proprio collegamento da remoto dove viene indicato che è valida per ogni tipo di installazione! Ora essendomi stato detto per un altro argomento che la guida che seguivo era apposita per Debian e non per ubuntu, volevo gentilmente sapere se troverò qualche differenza nelle procedure/comandi da impartire,

**Reply by snakuzzo** [reply]
Continui a non specificare bene la guida. Con "Collegarsi da remoto..." ci sono almeno 4 articoli.
Pretendo troppo se ti chiedi di riportare qui il titolo "per intero" o i link della guida a cui fai riferimento ?

Grazie.

**Reply by efestopini** [reply]
1. non capisco che problemi provochi usare "Add a comment" che è la normalità in ogni forum di mia conoscenza, ma se è utile lo faccio senza problem
  2. mi spiace se ti ho dato l'impressione di non aver letto quello che hai scritto, mi sembrava di aver risposto ai tuoi quesiti
la guida che vorrei seguire (visto che come ho scritto non ho ancora cominciato) è 
https://indomus.it/guide/collegarsi-da-remoto-a-home-assistant-via-reverse-proxy/
sapendo questo, e che praticamente tutte le guide riportano il messaggio messo bene in evidenza che sono valide per qualsiasi tipo di installazione, l'unica discriminante potrebbe essere il sistema operativo.
Quindi chiedo nuovamente, esiste qualche ragione per qui seguendo la guida potrei trovare difficoltà usando io Linux Ubuntu? 
Per favore, trattami pure senza problema come uno stupido totale (questo sono in questo campo) e se per rispondere alla mia domanda ti serve sapere qualcosa di specifico, che a te sembra banale, dimmelo chiaramente, probabilmente io non ci arrivo, in fondo seguo queste guide perché non lo so fare, se avessi le tue competenze non mi servirebbero delle guide, lo farei e basta.
Grazie

**Reply by snakuzzo** [reply]
1. Leggi bene quello che ti ho scritto e cerca la differenza tra "Add a comment" e "Add a Reply" qui su GitHub. Ogni discussione è organizzata per avere più thread. Se non noti la differenza, io non posso farci nulla. E, per inciso, questo è Discussions di GitHub, che viene usato a mo di forum, ma non è un classico forum.

2. Io non voglio e non ti ho trattato da stupido. Però, come ti ho già detto, se vieni e non indichi di preciso il titolo o il link della guida, io posso pensare qualsiasi cosa. Ti basta cercare sul sito quello che hai scritto nel primo post e capirai da solo il perché della mia insistenza nel chiederti informazioni precise. Tornando alla "sostanza", sì...va bene lo stesso

**Reply by efestopini** [reply]
grazie per la precisazione sul primo punto e grazie mille per la risposta alla mia domanda, io voglio essere considerato uno stupido, perché lo sono in questo campo, capisco che questo sito/forum/comunità sia frequentata da persone che ne sanno già molto del settore e che io sono probabilmente fuori posto, questo sito mi è stato consigliato appunto da uno del settore, mi rendo conto che probabilmente dovrei trovare un "posto" che tratti l'argomento in maniera più comprensibile per me.
Giusto per precisazione, se avessi posto lo stesso quesito ma in riferimento alle altre tre guide, la risposta sarebbe stata diversa?

**Reply by snakuzzo** [reply]
Non ho capito la domanda

**Reply by efestopini** [reply]
non sei naturalmente obbligato a rispondere ma mi incuriosiva sapere se seguendo la guida con l'acquisto del dominio (più semplice) o avessi io avuto necessità si seguire quelle per internet via nat, ( quella gratis o a pagamento) la tua risposta "Tornando alla "sostanza", sì...va bene lo stesso" sarebbe cambiata? Cioè, per esempio, la guida 

https://indomus.it/guide/collegarsi-da-remoto-a-home-assistant-via-reti-provider-che-usino-nat/ 

con installazione di assistant core su docker con sistema operativo linux ubuntu non sarebbe andata bene? avrei avuto problemi con passaggi che sarebbero stati diversi?

**Reply by snakuzzo** [reply]
Per quello specifico caso di installazione, che sotto ci sia Debian o Ubuntu, cambia nulla

**Reply by efestopini** [reply]
ok, quindi sarebbe giusto dire che se al mio post iniziale si fosse risposto, per esempio, "certo, che tu esegua un sistema operativo debian o ubuntu vai tranquillo che è uguale a meno che tu non abbia questa particolare situazione (non idea di quale potrebbe essere), in quel caso devi apportare questi cambiamenti. 
Io avrei risposto grazie mille, senza dilungare così la discussione.
Grazie mille

**Reply by snakuzzo** [reply]
Vabbè dai...mo basta però :D

**Reply by KC-inDomus** [reply]
> Io avrei risposto grazie mille, senza dilungare così la discussione.
> 
> Grazie mille

ma ringrazia tu che ti abbiamo risposto, piuttosto. e buon Natale.

**Reply by efestopini** [reply]
Se leggi bene quello che ho scritto e tra l'altro l'hai pure quotato, ho ringraziato, sono letteralmente le ultime due parole che ho scritto, la discussione si è dilungata per quello che sembra una pignoleria inutile alla risposta della mia domanda, sembra un guardare dall'alto uno che ne sa poco, e ho visto questo atteggiamento in altre risposte in altre domande di altri utenti che come me chiedevano cose per voi semplici, atteggiamenti come " non leggi quello che ti ho scritto, mi prendi in giro" e cose così, nella mia domanda iniziale per l'appunto non era necessario trattarmi così come stai facendo pure te tra l'altro, ma rispondere appunto vai tranquillo non cambia niente.
Ringrazio ancora e buon natale anche a te

**Reply by efestopini** [reply]
Se poi qui dentro do fastidio basta dirlo e non chiederò più niente

**Reply by KC-inDomus** [reply]
> Se poi qui dentro do fastidio basta dirlo e non chiederò più niente

vedi, in quasi sei anni di divulgazione a testa bassa lo staff ha empiricamente riconosciuto tre categorie di utenti: quelli che passano, prendono e se ne vanno in silenzio, quelli che passano, prendono e ringraziano e infine quelli che passano, prendono e ti spiegano come avresti dovuto servirli.
ecco: quest'ultima categoria personalmente non la posso vedere.

fai tu.

**Reply by efestopini** [reply]
Esistono anche quelli che passano, prendono, chiedono aiuto e vengono trattati con sufficienza e poi nonostante tutto ringraziano, io rientro in questa categoria visto che sono stato accusato di non aver neanche letto cosa mi è stato scritto, cosa non vera (sufficienza) e poi quando ho ricevuto la risposta ho ringraziato e non capisco come si possa dire il contrario visto che è ancora scritto la.

**Reply by KC-inDomus** [reply]
sei stato accusato a ragione di non aver letto. hai postato un problema senza fornire la benché minima indicazione, ti è stato fatto presente, hai risposto continuando a non dare indicazioni. cmq sia: se questo è l'approccio che hai, addio. c'è pieno di gruppi che ti accoglieranno a braccia aperte, qui si richiede di essere attenti e puntuali, ANCHE e sopratutto cercando di risolvere i propri problemi. auguri

**Reply by efestopini** [reply]
Magari ho letto e non ho capito, un pelo di comprensione per chi non conosce i termini magari la si concede, sopratutto se si premette di non essere del settore, io avevo letto e mi sembrava di aver dato tutto quello che serviva (cosa effettivamente confermata poi alla fine).

**Reply by KC-inDomus** [reply]
> e mi sembrava di aver dato tutto quello che serviva (cosa effettivamente confermata poi alla fine). 

no, non è così. sorry. quindi i consigli su come trattare chi frequenta questo forum, tielli per te.

**Reply by efestopini** [reply]
Hai ragione su tutti i fronti, se per caso avessi bisogno proverò a chiedere supporto qui, se mi sarà dato benissimo e ringrazierò, se mi verranno chieste cose che non sarò in grado fornire per mia incapacità ringrazierò ugualmente anche se non avrò risolto il problema, se non riceverò risposta pazienza, auguro a tutti I collaboratori, agli amministratori e a tutti gli utenti buon natale, e scusatemi ancora se sono così stupido da non capire concetti così semplici

**Reply by KC-inDomus** [reply]
> auguro a tutti I collaboratori, agli amministratori e a tutti gli utenti buon natale, e scusatemi ancora se sono così stupido da non capire concetti così semplici

non usare approcci aggressivo/passivi, ché non serve e non c'è bisogno. 
auguri anche a te.
